pkgs/applications/audio/aucatctl/default.nix

```
@@ -27,7 +27,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
 
   meta = with lib; {
-    description = "The aucatctl utility sends MIDI messages to control sndiod and/or aucat volumes";
+    description = "Aucatctl utility sends MIDI messages to control sndiod and/or aucat volumes";
```

Name of the utility probably should not be uppercased.

---

Thanks @jtojnar for chiming in on this, much appreciated.

According to the recommendation in [`pkgs.README`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes) the description should "[n]ot start with the package name", so it might be appropriate to remove it altogether. What do you think? I'll try to come up with a scripted way to address this recommendation too.


```suggestion
    description = "Utility sends MIDI messages to control sndiod and/or aucat volumes";
```

---

In this case having the program name as the start looks good to me. Replacing it with just "Utility" makes no sense.

---

> Utitility that allows sending MIDI Messages....

---

Any preference in terms of capitalisation, @mweinelt?